packages/compass-shell/package.json

```
@@ -54,6 +54,7 @@
   "peerDependencies": {
     "@leafygreen-ui/icon": "^11.4.0",
     "@leafygreen-ui/icon-button": "^9.1.5",
+    "@mongodb-js/compass-components": "*",
```

Having peer as `*` trips npm@7 every time you do another install in the package and it changes your prod/dev version to `*` which we really don't want to happen. We also don't need to do that `*` for peers, so I'd suggest to switch it to current version

```suggestion
    "@mongodb-js/compass-components": "^0.5.0"
```